importer/src/test/java/org/hiero/mirror/importer/downloader/block/simulator/BlockNodeSimulator.java

```diff
@@ -37,6 +37,7 @@ public final class BlockNodeSimulator implements AutoCloseable {
     private int port;
     private Server server;
     private boolean started;
+    private boolean withMissingBlock;
```

Comment:
Code smell: `Field withMissingBlock has the same name as a method`

```suggestion
    private boolean missingBlock;
```